app/client/create_survey_modal/create_survey_modal.jade

```diff
@@ -0,0 +1,25 @@
+template(name="create_survey_modal")
+  #create-survey-modal.modal.fade(tabindex="-1" role="dialog")
+    form.modal-dialog
+      .modal-content
+        .modal-header
+          button(type="button" class="close" data-dismiss="modal" aria-label="Close")
+            i.fa.fa-times-circle
```

Comment:
The new material design class names for this will be `mdi` and `mdi-close-circle`